shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -449,11 +449,11 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
   [accessibility_channel_.get() setMessageHandler:^(id message, FlutterReply reply) {
     HandleEvent((NSDictionary*)message);
   }];
-  _previous_route_ = nil;
```

Comment:
You should probably still initialize it to `nil` so it does not contain garbage (C++ is this way), but for better readability, do so in the initializer list.